src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/SessionService.kt

```diff
@@ -154,6 +159,14 @@ class SessionService(
     return emptyList()
   }
 
+  private fun getExcludeDates(prison: Prison): Set<LocalDate> {
+    val excludeDates = mutableSetOf<LocalDate>()
+    if (prison.excludeDates.isNotEmpty()) {
+      prison.excludeDates.forEach { excludeDates.add(it.excludeDate) }
+    }
+    return excludeDates.toSet()
```

Comment:
Will sorting this data have any performance advantages?

Reply:
not really we should think about performance when it's a problem